cmake/std/atdm/ride/environment.sh

```diff
@@ -26,8 +26,8 @@ else
   export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=32
 fi
 
+export ATDM_CONFIG_KOKKOS_ARCH=Power8,Kepler37
```

Comment:
@ibaned, what does the array syntax with `KOKKOS_ARCH` mean?  Is the first element `Power8` for the host CPU and `Kepler37` for the device/GPU?  If there is just one element, how is that interpreted with CUDA?  I can't find documentation that explains that.

Otherwise, this looks okay with me (w.r.t. to the configuration scripts).

Reply:
@bartlettroscoe The ordering of the items has no effect. If the CUDA backend is enabled, we strongly recommend including the GPU architecture in `KOKKOS_ARCH`, that will be used to add the `-arch=sm_??` flag to the command line and to select different low-level operations based on which NVIDIA architecture is being run on. If the GPU architecture is not specified, then no `-arch` flag will be added (`nvcc` will try to compile for all of them) and architecture-specific low-level ops may not be selected correctly.

Reply:
@bartlettroscoe I suspect there is no good error checking in that scenario, in fact you may end up in a situation where both GPU architectures are "enabled", and the code assumes that is not possible. We should add more robust error checking to prevent this.